src/containers/UserSettings/settings.ts

```diff
@@ -75,6 +76,16 @@ export const languageSetting: SettingProps = {
     },
 };
 
+export const binaryDataInPlainTextDisplay: SettingProps = {
+    settingKey: BINARY_DATA_IN_PLAIN_TEXT_DISPLAY,
+    title: i18n('settings.binaryDataInPlainTextDisplay.title'),
+    description:
+        process.env.REACT_APP_META_BACKEND === undefined ||
+        process.env.REACT_APP_META_BACKEND === 'undefined'
```

Comment:
Please do not use `REACT_APP_META_BACKEND` in common code. This is an env variable for our CRA. Let's make component like this
```
function Description() {
    const isSingleClusterMode = useTypedSelector((state) => state.singleClusterMode);
    if (isSingleClusterMode) {
        return null;
    }

    return <span>{i18n('settings.binaryDataInPlainTextDisplay.description')}</span>;
}

description: <Description />
```

Reply:
Added `src/containers/ClusterModeGuard/ClusterModeGuard.tsx` container for that.